tools/single_use/Normalize_Consent_Data.php

```diff
@@ -99,8 +99,8 @@
         $consentID  = $entry['ID'];
         $candID     = $entry['CandID'];
         // Check if consent status is not_answered
-        if($status === "not_answered") {
-            array_push($errors, "Deprecated consent status 'not_answered' found for " . $consentName . ":
+        if($status === "not_answered" || $status === "") {
+            array_push($errors, "Deprecated consent status 'not_answered' or empty string found for " . $consentName . ":
```

Comment:
I think these merit different error messages. `not_answered` is deprecated, empty string was never really valid in the schema, but MySQL's stupid handling of enums sometimes inserts it anyways.